counterexamples/admins/locality/invalid-max-limit-iso-sub-country-code.json

```diff
@@ -0,0 +1,46 @@
+{
+    "type": "Feature",
+    "properties": {
+      "theme": "admins",
+      "type": "locality",
+      "version": 0,
+      "updateTime": "2023-02-22T23:55:01-08:00",
+      "subType": "administrativeLocality",
+      "localityType": "country",
+      "adminLevel": 1,
+      "isoSubCountryCode": "US-NYCC",
+      "defaultLanguage": "english",
+      "drivingSide": "right",
+      "contextId": "northAmericaId",
+      "extExpectedErrors": [
+        "[I#/properties/defaultLanguage] [S#/$defs/propertyDefinitions/language/oneOf/1/pattern] does not match pattern '^[a-zA-Z]{2,3}(-[a-zA-Z]{4})?(-[a-zA-Z]{2})?$'"
+      ]
```

Comment:
This seems like the wrong error - isn't it looking at a failure in the `defaultLanguage` property?

1. [ ] Should drop `defaultLanguage` because it contains an error that is irrelevant to the example.
2. [ ] Should fix expected error...